src/App.tsx

```diff
@@ -1,18 +1,91 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useContext, useMemo, useState } from 'react';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodosContext } from './components/context/TodoContext';
+import { Todo } from './types/Todo';
+import { Status } from './types/StatusEnum';
 
 export const App: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [filterStatus, setFilterStatus] = useState(Status.All);
+
+  const todosContext = useContext(TodosContext);
+
+  if (!todosContext) {
+    return null;
+  }
+
+  const {
+    todos,
+    addTodo,
+    deleteCompletedTodos,
+  } = todosContext;
+
+  const incompletedTodosCount = todos.filter(
+    (todo: Todo) => !todo.completed,
+  ).length;
+
+  const hasCompletedTodos = todos.some(todo => todo.completed);
+
+  // #region #HANDLING EVENTS
+  function handleTitleChange(event: React.ChangeEvent<HTMLInputElement>) {
```

Comment:
I see you have arrow functions and function declarations. You need to keep consistency in your code. use only 1 of the options